docs/structured-streaming-programming-guide.md

```diff
@@ -2365,6 +2367,119 @@ When the streaming query is started, Spark calls the function or the object’s
   See [SPARK-28650](https://issues.apache.org/jira/browse/SPARK-28650) for more details.
   If you need deduplication on output, try out `foreachBatch` instead.
 
+#### Streaming Table APIs
+Since Spark 3.1, you can also use `DataStreamReader.table()` and `DataStreamWriter.toTable()` to read and write streaming DataFrames as tables:
```

Comment:
How about:

```suggestion
Since Spark 3.1, you can also use `DataStreamReader.table()` to read tables as streaming DataFrames and use `DataStreamWriter.toTable()` to write streaming DataFrames as tables:
```

Reply:
suggested ones looks more correct.